From 8691105e1808ec9b031b6c25f70204a0908ae9c1 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Sat, 1 Jun 2019 13:09:41 +0200
Subject: [PATCH] gtk-doc: do not include tree_index.sgml

gtk-doc 1.30 no longer generates the file if the object tree is empty

Upstream-Status: Backport [87d1add318b5e5d09977f7f374e923577b6ff3be]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 docs/libpsl/libpsl-docs.sgml | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/docs/libpsl/libpsl-docs.sgml b/docs/libpsl/libpsl-docs.sgml
index 1eca9a2..1c38dbc 100644
--- a/docs/libpsl/libpsl-docs.sgml
+++ b/docs/libpsl/libpsl-docs.sgml
@@ -22,10 +22,6 @@
         </para>
         <xi:include href="xml/libpsl.xml"/>
   </chapter>
-  <chapter id="object-tree">
-    <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
-  </chapter>
   <index id="api-index-full">
     <title>API Index</title>
     <xi:include href="xml/api-index-full.xml"><xi:fallback /></xi:include>
